Jenkinsfile

```diff
@@ -3,59 +3,99 @@ properties(
     parameters(
       [
         string(
-          defaultValue: 'x86_64,ppc64le',
+          defaultValue: 'x86_64',
           description: 'A comma separated list of architectures to run the test on. Valid values include [x86_64, ppc64le, aarch64, s390x].',
           name: 'ARCHES'
-        )
+        ),
+        string(
+            defaultValue: 'SSHPRIVKEY',
+            description: 'SSH private key Jenkins credential ID for Beaker/SSH operations',
+            name: 'SSHPRIVKEYCREDENTIALID'
+        ),
+        string(
+            defaultValue: 'SSHPUBKEY',
+            description: 'SSH public key Jenkins credential ID for Beaker/SSH operations',
+            name: 'SSHPUBKEYCREDENTIALID'
+        ),
       ]
     )
   ]
 )
 
-@Library('multiarch-ci-libraries@dev') _
+List arches = params.ARCHES.tokenize(',')
 
-import com.redhat.multiarch.ci.Slave
+library changelog: false, identifier: 'multiarch-ci-libraries@dev', retriever: legacySCM([$class: 'GitSCM', branches: [[name: '*/dev']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[url: 'https://github.com/scoheb/multiarch-ci-libraries']]])
 
-def List arches = params.ARCHES.tokenize(',')
-def Boolean runOnProvisionedHosts = true;
-def Boolean installAnsible = true;
+// Populate the Provisioning Config
+def config = provisioningConfig.create()
```

Comment:
I love this new configuration object style - it keeps the call into the library as simple as possible.

What I dislike, however, is that this new objects lives in the middle of the default dummy test template. I think what we should do is create a default config that lives somewhere that we can reference - maybe a provisioner.config or even something living in the shared lib. This way, the template stays as simple as possible. I think it would be best for the defaults for that to be what would work for Martin's Jenner's folks using my system.

Maybe we could look into added a contra.config to the shared lib for you guys - or even creating a contra-test-template repo / branch.